spec/spec_helper.rb

```diff
@@ -0,0 +1,8 @@
+# encoding: utf-8
+
+require 'aws-sdk'
+require 'logstash/devutils/rspec/spec_helper'
+require 'logstash/outputs/sqs'
+
+require 'logstash/logging/logger'
```

Comment:
@joshuaspence explicit is usually prefered (in our code base) but since this is the logger I would remove it.